packages/widgetbook/README.md

```diff
@@ -147,48 +215,68 @@ class HotReload extends StatelessWidget {
 }
 ```
 
-## Set Widgetbook name
+## `appInfo`
 
-Customize `Widgetbook`'s name according to the project by using appInfo:
+The `appInfo` property allows users to label the `Widgetbook` in case you are maintaining more than one `Widgetbook`'s for multiple projects. Customize `Widgetbook`'s name according to the project by using appInfo:
 
 ```dart
-Widgetbook(
+Widgetbook.material(
   appInfo: AppInfo(
     name: 'Your apps name',
   ),
 )
 ```
 
-## Adding Themes
+## Localization
+
+Widgetbook defines the two properties `supportedLocales` and `localizationsDelegates` to support localization of `Widget`s. These values behave as described in [Flutter Internationalization](https://docs.flutter.dev/development/accessibility-and-localization/internationalization).
 
-Import your app's theme for a realistic preview by using `Widgetbook`'s `lightTheme` and `darkTheme` properties:
 ```dart
-Widgetbook(
-  lightTheme: lightTheme,
-  darkTheme: darkTheme,
+Widgetbook.material(
+  localizationsDelegates: [
+    GlobalMaterialLocalizations.delegate,
+    GlobalWidgetsLocalizations.delegate,
+    GlobalCupertinoLocalizations.delegate,
+  ],
+  supportedLocales: [
+    Locale('en'), // English, no country code
+    Locale('es'), // Spanish, no country code
+  ],
 )
 ```
 
-A preview in Widgetbook's light or dark mode is only shown when the appropriate theme is set. 
+## `themes`
 
-If you are in development and haven't defined a theme yet, you can use `ThemeData.light()` or `ThemeData.dark()` to use a default theme.
+Import your app's theme for a realistic preview by using `Widgetbook`'s `theme` property:
+```dart
+Widgetbook.material(
+   themes: [
+    WidgetbookTheme(
+      name: 'Light',
+      data: ThemeData.light(),
+    ),
+    WidgetbookTheme(
+      name: 'Dark',
+      data: ThemeData.dark(),
+    ),
+  ],
+)
+```
 
-## Add devices
+## `devices`
 
 Customize the preview by defining preview devices: 
 
 ```dart
-Widgetbook(
+Widgetbook.material(
   devices: [
     Apple.iPhone11,
-    Apple.iPhone12,
-    Samsung.s10,
     Samsung.s21ultra,
   ]
 )
 ```
 
-Right now there is a predefinied short list of devices but let us know which you need in our [Discord](https://discord.gg/zT4AMStAJA). We will extend the list of predefinied devices in the future!
+Right now there is a predefinied list of devices but let us know which you need in our [Discord](https://discord.gg/zT4AMStAJA).
```

Comment:
Right now,

Reply:
Right now, there is a predefined list of devices. If you need more devices, you can either add them on your own or let us know which ones you need in our ...